src/test/rmc/Never/never_return.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![feature(never_type)]
+
+/// Test using the never type
+pub fn err() -> ! {
+    panic!("EXPECTED FAIL: Function should always fail");
+}
+
+// Give an empty main to make rustc happy.
+#[no_mangle]
+pub fn main() {
+    //let var = rmc::nondet::<i32>();
```

Comment:
why can't we do nondet

Reply:
oops... we can. I modified this test while reading the MIR and I forgot to change it back. Let me fix it.